src/Utilities/mockoptimizer.jl

```diff
@@ -19,7 +19,9 @@ mutable struct MockOptimizer{MT<:MOI.ModelLike} <: MOI.AbstractOptimizer
     conattribute::Dict{MOI.ConstraintIndex,Int} # MockConstraintAttribute
     needsallocateload::Bool # Allows to tests the Allocate-Load interface, see copy!
     canaddvar::Bool
-    canaddcon::Bool # If false, the optimizer throws CannotAddConstraint
+    canaddcon::Bool # If false, the optimizer throws AddConstraintNotAllowed
```

Comment:
Nit: For consistency these should be named like `add_con_allowed` instead of `canaddcon`.